docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -2,9 +2,11 @@
 Submitting contributions
 ========================
 
-We're always grateful for contributions to Django's code. Indeed, bug reports
-with associated contributions will get fixed *far* more quickly than those
-without a solution.
+We are always grateful for contributions to Django's code.
+To ensure a smooth process, please create a ticket for :ref:`non-trivial-contribution`
+on our `ticket tracker`_ before submitting your contribution.
```

Comment:
I'm tempted for us to not mention tickets yet and have something like
```
Thank you for your interest in contributing to Django's code.
```
(can be bigger mentioning that Django is run on the effort of volunteers and these docs are to help you understand Django's processes and land your contribution successfully)